sdk/nodejs/runtime/closure/parseFunction.ts

```diff
@@ -493,32 +501,49 @@ function computeCapturedVariableNames(file: ts.SourceFile): CapturedVariables {
         // We want to capture a specific set of properties.  Add this set of properties
         // into the existing set.
         const combined = existing || [];
-
-        // See if we've already marked this property as captured.  If so, make sure we still record
-        // if this property was invoked or not.
-        for (const existingProp of combined) {
-            if (existingProp.name === current.name) {
-                existingProp.invoked = existingProp.invoked || current.invoked;
```

Comment:
i removed this merging logic as it was too complex now that we have 'chains' of property-accesses recorded.  Instead, now we just record all the chains, and we consider all of them (including dupes) when doing closure serialization.  so if we have  "a.b.c()" and "a.b.c" and "a.b.c", we'll record three chains of 'b.c' (with the first saying it is invoked).  Then, in closure serialization, when we're emitting the value for 'c', we'll see if any of the accesses were 'invoked'.